Exiled.API/Extensions/Item.cs

```diff
@@ -316,14 +316,7 @@ public static void SetOther(this Player player, Inventory.SyncItemInfo weapon, O
         /// <returns>Returns the item durability.</returns>
         public static float GetDefaultDurability(ItemType item)
```

Comment:
Shouldn't it be `this ItemType item` so that it may be used as an extension?

Reply:
Yes, I agree.